src/senaite/sync/browser/views.py

```diff
@@ -94,18 +87,34 @@ def __call__(self):
 
         # Handle "Fetch" action
         if form.get("fetch", False):
+
+            url = form.get("url", "")
+            if not url.startswith("http"):
+                url = "http://{}".format(url)
+            domain_name = form.get("domain_name", None)
+            username = form.get("ac_name", None)
+            password = form.get("ac_password", None)
             # check if all mandatory fields have values
-            if not all([self.domain_name, self.url, self.username,
-                        self.password]):
+            if not all([domain_name, url, username,
+                        password]):
                 message = _("Please fill in all required fields")
                 self.add_status_message(message, "error")
                 return self.template()
 
+            content_types = form.get("content_types", None)
+            if content_types is not None:
+                content_types = [t.strip() for t in content_types.split(",")
+                                 if t]
```

Comment:
I think you can just do a `[t.strip() for t in content_types.split(",")]` since you filter out invalids below